jparest/pom.xml

```
@@ -77,6 +77,12 @@
             <artifactId>hibernate-validator</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.10.1</version>
```

We should add the test scope here as we only need this dependency for tests atm.
```suggestion
            <version>2.10.1</version>
            <scope>test</scope>
```

---

Done